src/features/cartographie/layouts/cartographie/cartographie.layout.html

```diff
@@ -155,30 +155,32 @@ <h3 class="card-title text-primary">
             target="_blank"
             rel="noopener noreferrer">
             <span class="ri-circled bg-primary text-light fw-normal me-3" aria-hidden="true">
-              <i class="ri-calendar-todo-line" aria-hidden="true"></i>
+              <span role="img" class="ri-calendar-todo-line" aria-hidden="true"></span>
             </span>
             <span class="orientation-btn-label my-auto text-start lh-sm">
               Participez au(x) webinaire(s)
               <br />
-              <i class="small text-muted">
+              <span role="img" class="small text-muted">
                 Mettre à jour ses données, utiliser l'outil d'orientation, nous vous proposons des webinaires pour vous
                 accompagner dans l'utilisation de la cartographie
-              </i>
+              </span>
             </span>
           </a>
           <a
             class="btn orientation-btn bg-white mb-3 p-3 rounded-0 d-flex d-flex align-items-center"
             href="https://labase.anct.gouv.fr/base/356?tags=&page=0&tab=resources"
             target="_blank"
             rel="noopener noreferrer">
-            <span class="ri-circled bg-primary text-light fw-normal me-3" aria-hidden="true"><i class="ri-link"></i></span>
+            <span role="img" class="ri-circled bg-primary text-light fw-normal me-3" aria-hidden="true">
+              <span role="img" class="ri-link"></span>
```

Comment:
```suggestion
              <span class="ri-link"></span>
```
Je pense que tu peux exceptionnellement supprimer `role="img"` ici, car il est présent dans l'élément parent